FreeRTOS/iot-reference-arm-corstone3xx
# Consolidate common CMake configurations

Consolidate common CMake configurations

Description
-----------
Move common settings and values from the applications to the bsp and root CMakeLists.txt files. These values are common across all apps.

Test Steps
-----------
All platforms are tested with all the examples.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
